tests/metaworld/envs/mujoco/sawyer_xyz/utils.py

```diff
@@ -1,14 +1,18 @@
-def check_success(env, policy, render=False):
-    """Tests whether a given policy solves an environment
+import numpy as np
+
 
+def check_success(env, policy, noisiness, render=False):
+    """Tests whether a given policy solves an environment
     Args:
         env (metaworld.envs.MujocoEnv): Environment to test
         policy (metaworld.policies.policies.Policy): Policy that's supposed to succeed in env
+        noisiness (float): Decimal value indicating std deviation of the noise as a % of action space
```

Comment:
The param is named `noisiness` in PR #92 as well, but @ryanjulian wants it renamed to `action_std_pct`. It doesn't matter whether we fix it here or there, but we'll need to be sure to rebase on master after one of the PR's gets merged.